src/actions/icons.js

```diff
@@ -56,7 +56,10 @@ export function iconStartLoading(icon: string): ThunkAction<Promise<void>> {
           dispatch(iconIsInError(icon));
           break;
         case 'cached':
-        // nothing to do
+          // nothing to do
+          break;
+        default:
+          throw new Error(`Unknown icon load result ${result}`);
```

Comment:
I considered removing the  `default-case` rule, but then I noticed that Flow doesn't warn if we don't handle all possible cases in a switch... so the rule is still useful. If Flow ever checks this we can consider removing the rule.